GitCommands/Git/Executable.cs

```diff
@@ -78,15 +78,15 @@ private sealed class ProcessWrapper : IProcess
                                   bool redirectOutput,
                                   Encoding? outputEncoding,
                                   bool useShellExecute,
-                                  bool throwOnErrorOutput)
+                                  bool throwOnErrorExit)
```

Comment:
Other occurrences of this function argument can be renamed throughout the project. in a follow-up. I guess it would cause too many MCs for pending PRs.

Reply:
I suggest that this is done when the PR is approved and is to be merged.